misc/dbt-materialize/dbt/include/materialize/macros/mz_create_index.sql

```diff
@@ -0,0 +1,49 @@
+-- Copyright Materialize, Inc. and contributors. All rights reserved.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License in the LICENSE file at the
+-- root of this repository, or online at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+{% macro mz_create_index(obj_name, default=False, idx_name="", col_refs=[], with_options=[]) -%}
```

Comment:
it's usually a bad idea to use lists as default arguments to methods in python ([explanation](https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html)), I don't know if dbt macros have some magic around this but it's more future-proof to set these to none and explicitly check for that.